### ISSUE TITLE
GitHub Sponsor

### ISSUE THREAD

**Issue by ThomasGorisse** [betagouv/mon-entreprise#1773]
Bonjour,

Tout d'abord un grand merci pour la plateforme très utile et très bien faite.

Serait-t-il possible d'ajouter une section concernant le sponsoring et notamment GitHub Sponsor ?
Je ne trouve aucune information là dessus et les revenus de la plateforme étant en général plus de l'ordre du "Buy me coffee", je me demande si cela doit impliquer la création d'un statut d'auto entrepreneur ou si la plateforme GitHub envoie directement les informations pré remplies au service de déclaration d'impôts sur le revenu sans avoir besoin d'un statut spécifique.

Merci

**Comment by johangirod** [comment]
Effectivement, il semblerait que cela soit considéré comme un revenu d'activité, et doivent être déclaré comme tels : 

Sources : 
- https://tipeee.zendesk.com/hc/fr/articles/360010436039-Dois-je-d%C3%A9clarer-les-fonds-collect%C3%A9s-sur-Tipeee-
- https://blog.wizdeo.com/fr/55-general/1201-fiscalite-des-dons-youtube-superchat-tipee-twitch-etc

A ajouter à l'assitant économie collaborative "que dois-je déclarer"
